app/dictionary.en-us.words

```diff
@@ -10996,6 +11008,7 @@ nowadays
 noway
 nowhere
 nowt
+ntfs
```

Comment:
Thanks!, Hey please move this to 
ci_edit/app/dictionary.acronyms.words

P.S. I've been wondering about suggesting that acronyms be capitalized (since that would be proper). To keep that option open, I'm trying to keep the acronyms separate. Trivia, I'm also aware that many of them are actually initialisms and not acronyms; sorry if that's confusing.

Reply:
Sure thing!